distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -480,6 +486,214 @@ public void testBasicRestAccess() throws Exception {
         getSecurityPolicy().waitForGuestAuthReady(url);
     }
 
+    @Test(expected = NoHttpResponseException.class)
+    public void testBasicAuthOverHttp() throws Exception {
+        String url = SERVICE_ROOT.getUrl() + "/catalog/query?q=*&src=local";
+
+        //try basic auth over http
+        try {
+            configureRestForBasic(SDK_SOAP_CONTEXT);
+            waitForSecurityHandlers(url);
+
+            given().auth()
+                    .basic("admin", "admin")
+                    .when()
+                    .get(url.replace("https://", "http://"));
+        } finally {
+            configureRestForGuest(SDK_SOAP_CONTEXT);
```

Comment:
I guess I was just wondering if it would hurt anything if we would always call `configureRestForGuest` after all tests in the `@After` method? That way we wouldn't need the `finally` and forget to call it.